Raven.NET.Core/Observers/RavenTypeWatcher.cs

```diff
@@ -0,0 +1,13 @@
+using Raven.NET.Core.Observers.Interfaces;
+using Raven.NET.Core.Subjects;
+
+namespace Raven.NET.Core.Observers
+{
+    public class RavenTypeWatcher : IRavenTypeWatcher
+    {
+        void IRaven.Update(RavenSubject subject)
```

Comment:
Explicit modifier would make code easier to read

Reply:
You can not set explicit modifer becouse this is not direct implementation of interface

Reply:
In that case it's fine